ui/src/linking/linking-store.ts

```diff
@@ -25,13 +25,14 @@ import {
 import { GeometryKmPostId } from 'geometry/geometry-model';
 import { angleDiffRads, directionBetweenPoints, interpolateXY } from 'utils/math-utils';
 import { exhaustiveMatchingGuard, expectDefined } from 'utils/type-utils';
+import { draftLayoutContext } from 'common/common-model';
 
 export const linkingReducers = {
     startAlignmentLinking: (
         state: TrackLayoutState,
         { payload }: PayloadAction<GeometryPreliminaryLinkingParameters>,
     ): void => {
-        state.publishType = 'DRAFT';
+        state.layoutContext = draftLayoutContext(state.layoutContext);
```

Comment:
Tässä hyvähkö esimerkki siitä miten nämä vaikutti paikannuspohjaan. Aiemmin ollaan fiksattu että halutaan `DRAFT`, ja niin halutaan edelleenkin. Täällä ei kuitenkaan haluta ottaa kantaa `designId`:hen, niin käytetään konteksti funktion läpi joka fiksaa `publicationState`:n `DRAFT`:iksi, mutta jättää `designId`:n rauhaan

Reply:
Aa aivan, meinasit näitä kohtia. Joo, näissä voisi hyvin vain asettaa tuon pelkän `PublicationState`:n. Taidan muuttaa sen niin, jolloin se myös toiminee samalla tavalla kuin aiemmin